package/samplers/mocma/mocma.py

```diff
@@ -0,0 +1,395 @@
+from collections.abc import Sequence
```

Comment:
This sampler does not work with Python 3.9 without future annotations.

```suggestion
from __future__ import annotations

from collections.abc import Sequence
```

Reply:
Thank you for your comment. I applied your suggestion.